text/0000-project-goals.md

```diff
@@ -0,0 +1,196 @@
+* **To everything, there is a season.** While there will be room for accepting new goals that come up during the year, we primarily want to pick goals during a fixed time period and use the rest of the year to execute.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+The long-term vision is to create a sustainable goals process for the project. Per the axiom that **goals cover a problem, not a solution**, this RFC does not propose a specific process. **Rather, the goal is to devise the process.** To help us get going quickly, the intent is that the goal owner will design and drive an experiental process, including (a) selecting a slate of goals that will be confirmed by the teams they affect; (b) monitoring and reporting on progress towards those goals; and (c) developing infrastructure to support that monitoring and lessen the load. Experience from that will be used to shape an RFC that describes the process to use for the future (assuming the experiment is a success). 
+
+## Ownership and other resources
+
+**Owner:** nikomatsakis
+
+* nikomatsakis can commit 20% time (avg of 1 days per week) to pursue this task, which he estimates to be sufficient.
+
+### Support needed from the project
+
+* Project website resources to do things like
+    * post blog posts on both Inside Rust and the main Rust blog;
+    * create a tracking page (e.g., `https://rust-lang.org/goals`);
+    * create repositories, etc.
+* For teams opting to participate in this experimental run:
+    * they need to meet with the goal committee to review proposed goals, discuss priorities;
+    * they need to decide in a timely fashion whether they can commit the proposed resources
+
+## Outputs and milestones
+
+### Outputs
+
+There are three specific outputs from this process:
+
+* A **goal slate** for the second half of 2024, which will include
+    * a set of goals, each with an owner and with approval from their associated teams
+    * a high-level write-up of why this particular set of goals was chosen and what impact we expect for Rust
+* **Regular reporting** on the progress towards these goals over the course of the year
+* An **RFC with a finalized process** that we can use going forward
+
+### Milestones
+
+Key milestones along the way (with the most impactful highlighted in bold):
+
+| Date | Milestone |
+| --- | --- |
+| **Apr 26** | **Kick off the goal collection process** |
+| May 24 | Publish draft goal slate, take feedback from teams |
+| June 14 | Approval process for goal slate begins |
+| **June 28** | Publish final goal slate |
+| July 15 | Open RFC with future goals process |
```

Comment:
This seems too early - I doubt we would get enough feedback on the process to know if its successful (and to have a well-guided RFC).

Reply:
That makes sense to me, perhaps one of the anticipated goals for the next slate (2025 H1) can be an RFC.